webpack/scenes/ContentViews/Details/DetailsContainer.js

```diff
@@ -31,11 +31,6 @@ const DetailsContainer = ({ children, cvId, isOpen }) => {
 DetailsContainer.propTypes = {
   children: PropTypes.element.isRequired,
   cvId: PropTypes.number.isRequired,
-  isOpen: PropTypes.bool,
-};
-
-DetailsContainer.defaultProps = {
-  isOpen: true,
```

Comment:
What was this isOpen prop about in the first place? It's not a modal dialog... It might be a pre-Andrew-ism :)